packages/opentelemetry-basic-tracer/src/Span.ts

```diff
@@ -110,17 +111,24 @@ export class Span implements types.Span {
     return true;
   }
 
-  toString() {
-    const json = JSON.stringify({
+  toReadableSpan(): ReadableSpan {
+    return {
```

Comment:
Based on previous discussions, this should simply be `return this`. The idea being that the same object is received everywhere including exporters, but what you can do with the object changes based on the interface.